docs/css/codemirror.css

```diff
@@ -122,7 +122,7 @@ div.CodeMirror span.CodeMirror-nonmatchingbracket {color: #f22;}
 }
 
 /* The fake, visible scrollbars. Used to force redraw during scrolling
-   before actuall scrolling happens, thus preventing shaking and
+   before actual scrolling happens, thus preventing shaking and
```

Comment:
This is third-party code, which I don't think we should modify (you can submit a PR to them to fix it https://github.com/codemirror/CodeMirror/blob/master/lib/codemirror.css#L168)